eurocodedesign/geometry/steelsections/__init__.py

```diff
@@ -240,7 +241,7 @@ def _get_section_type(section_name: str) -> str | None:
     return None
 
 
-def _load_section_props(section_name: str) -> pd.Series:
+def _load_section_props(section_name: str) -> Any:
```

Comment:
Why does this need to change to `Any`? is `pd.Series[float]` not suitable?

Reply:
pd.Series[float] type checking is not fully supported and leads to errors.